### PR TITLE
Removed redundancy issue (#77)

### LINKED ISSUE #77: remove content redundancy between desktop and mobile
It is bad practise to have redundant code or content. please remove SupporterMobile.js and merge the logic into a single file that is just context aware regarding the graphics

### PR DESCRIPTION
**Pull request by Alirazadar**
closes #77

### CODE REVIEW

File: src/SupporterReward.js [hunk 1]
```diff
@@ -21,83 +23,84 @@ export default function SupporterReward (props) {
...
+                                               
+
+                                                <Grid.Column id="mobilesupport" style={{ bottom: '9.188rem', position: 'absolute' }}>
+                                           
+
+                                                        <div className="">
+                                                        <span style={{ color: '#000065' }} >Scenario 2: ≥ 40,000 KSM bonded</span>
+                                                        <span style={{ color: '#70e6da', fontSize: '15px' }} >40,000 TEER distributed to all contributors</span>
+
+                                                        If Integritee doesn’t win an auction and the crowdloan ends
```
Comment (brenzi) [inline src/SupporterReward.js:87]: why is this still redundant?

**Review by brenzi — CHANGES_REQUESTED** [review]

File: src/SupporterReward.js (hunk 1 above)
Comment (brenzi) [reply src/SupporterReward.js:87]: the same text appears twice in this file

Comment (brenzi) [reply src/SupporterReward.js:87]: I'm totally wrong, sorry. not the same text. my bad

**Comment by brenzi** [comment]
@mosonyi PR is fine for me, but I haven't tested. can you?

File: src/SupporterReward.js (hunk 1 above)
Comment (Alirazadar) [reply src/SupporterReward.js:87]: @brenzi thank you for the clarification!

**Comment by anizeani** [comment]
tested on mobile and it works. 
2 remarks: 
1. Couldn't test the contribution vis polkadot js extension, because it is not available for mobile, just for chrome & firefox on computer, right? Is there another way to test contribution via mobile or just not possible? (maybe add it as a 3rd hint after "ensure that you have unbounded KSM ...". Of course only if you think it's relevant
2. The hamburger button in the upper menu/navbar: when pressing it and then pressing on a topic, the hamburger window remains open until I repress the hamburger button. Is this the desired behaviour? Usually pressing a topic should close the hamburger window immediately..

**Review by brenzi — APPROVED** [review]

**Comment by brenzi** [comment]
mobile is fine, but desktop not, grphic should not cover text and dots should be close to milestone text

**Comment by anizeani** [comment]
Different screen sizes are shown here. 
1. The graphic never overlaps with the text (different screen sizes are shown. The developer tools in chrome are used to display the different screen resolutions (1920x1080, 1600x900, 1680x1050, 1300x768 and below 1200width (mobile--> folds text and removes background image)
2. The 2 dots are close to the text

An easy way to verify/test it is: 
on pc: yarn start --> in chrome open console and move the slider to make screen smaller or use the actual developer tools to set screen size / device size.
on real mobile: in the same wifi network go to the pc's ip adress:port running the react project
Example images:  
below_1200px_width_without_image_(mobile_view)

**Review by brenzi — APPROVED** [review]

**Pull request merged**